GraphQL-TSC.md

```diff
@@ -162,24 +168,43 @@ When something cannot be decided by consensus or a formal vote is required, the
 
 The ballots may be public or private depending upon the situation, but the aggregate results should always be published.
 
-#### Minimum threshold for votes
+### Defining a quorum of "attending" members
+
+To balance preserving the voting ability of all TSC members with the desire for the voting process to remain efficient and avoid deadlocks, we define a valid quorum based on members' recent attendance of GraphQL Working Group meetings.
 
-Because we work in a distributed environment, the voting process must account for a range of time zones and schedules. A quorum of at least half (or supermajority when relevent) of the TSC voting members must cast a ballot in order for the vote to be valid. Once the voting threshold has been met, one of these two critera must be satisfied to conclude the vote:
+A quorum is a majority (more than half, or 2/3 for a supermajority vote) of the TSC *attending members*. A quorum must cast a ballot in order for a vote to be valid.
+
+A TSC *attending member* is a member who has attended one of the previous three meetings. Should a TSC member miss three consecutive meetings, they will no longer be counted when determining quorum (but may still vote). A member starts counting towards quorum as of attending a meeting.
+
+### Voting process
+
+Because we work in a distributed environment, the voting process must account for a range of time zones and schedules. Once the threshold of a quorum has been met and a vote is valid, one of these two critera must be satisfied to conclude a vote:
 
 * A notice is sent via email that the vote will conclude in three business days, reminding those who haven't voted that they should do so. The vote will conclude at the end of this time.
-* The election results will not change if additional TSC members vote.
+* The election results would not change with the addition of another vote.
```

Comment:
I’m not comfortable with this, why is there a one vote threshold? Maybe 
```suggestion
* The election results would not change if all remaining TSC *attending members* (if any) were to vote.
```

Reply:
I think your intent is the same as what's here. We could make it more clear.

Eg for a majority vote, as soon as you get 6 yes votes (of 11 TSC) then there's no need to wait any longer for more people to vote. 

However you wouldn't want this to be based on attending membership since you want to allow room for others to vote. 

In other words, immediately cease collecting votes as soon as the next vote could not possibly affect the result

Reply:
Right; but imagine you have:

```
Yes: 4
No: 2
```

The addition of another vote (`no`) would not change the result (4 yes/3 no); however the addition of _three_ more votes (three "no"s) could change the result: (4 yes/5 no).

My issue was with "another" taken to mean a singular vote.

I'm totally happy with `if all remaining members were to vote` rather than just the attending members, I was just guessing at your intent.